noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -356,58 +370,556 @@
+                    FROM arkivstruktur.registrering r
+                    WHERE r._ref_mappe IS NULL
+                    GROUP BY r._ref_arkivdel, r._ref_klasse;
+                ]]>
+            </info>
+        </queries>
+    </check>
+
+    <check id="ASC9">
+        <title>Records without document descriptions</title>
+        <description>Number of records without any Document Descriptions grouped by Series</description>
+        <group>arkivstruktur</group>
+        <queries>
+            <info>
+                <![CDATA[
+                    SELECT r._ref_arkivdel series_system_id, COUNT(*) records_without_documents
+                    FROM arkivstruktur.registrering r
+                    WHERE r.systemid NOT IN (SELECT _ref_registrering FROM arkivstruktur.dokumentbeskrivelse WHERE _ref_registrering IS NOT NULL GROUP BY _ref_registrering)
+                    GROUP BY r._ref_arkivdel;
+                ]]>
+            </info>
+        </queries>
+    </check>
+
+    <check id="ASC10">
+        <title>Document Description count</title>
+        <description>Number of Document Descriptions grouped by Series</description>
+        <group>arkivstruktur</group>
+        <queries>
+            <info>
+                <![CDATA[
+                    SELECT r._ref_arkivdel series_system_id, count(*) number_of_document_description
+                    FROM arkivstruktur.dokumentbeskrivelse d
+                    JOIN arkivstruktur.registrering r ON r.systemid = d._ref_registrering
+                    GROUP BY r._ref_arkivdel
+
+                    UNION ALL
+
+                    SELECT 'total', count(*)
+                    FROM arkivstruktur.dokumentbeskrivelse;
+                ]]>
+            </info>
+        </queries>
+    </check>
+
+    <check id="ASC11">
+        <title>Document Descriptions without Document Objects</title>
+        <description>Number of Document Descriptions grouped by Series</description>
+        <group>arkivstruktur</group>
+        <queries>
+            <info>
+                <![CDATA[
+                    SELECT r._ref_arkivdel series_system_id, COUNT(*) document_descriptions_without_document_objects
+                    FROM arkivstruktur.dokumentbeskrivelse d
+                    JOIN arkivstruktur.registrering r ON r.systemid = d._ref_registrering
+                    WHERE d.systemid NOT IN (
+                        SELECT _ref_dokumentbeskrivelse
+                        FROM arkivstruktur.dokumentobjekt
+                        WHERE _ref_dokumentbeskrivelse IS NOT NULL
+                        GROUP BY _ref_dokumentbeskrivelse
+                    )
+                    GROUP BY r._ref_arkivdel;
+                ]]>
+            </info>
+        </queries>
+    </check>
+
+    <check id="ASC12">
+        <title>Document Object count</title>
+        <description>Number of Document Objects grouped by Series and parent type (Document Description or Record)</description>
```

Comment:
`Number of document objects grouped by series and parent (document description or record)`